### DISCUSSION TITLE
Portuguese (Português)

### DISCUSSION THREAD

**Discussion by OsodracPT** [sovenor/bitcoin-rocks#13] [Translations]
Hey! I have started to work on the portuguese translation.
So far completed  `404_pt`, `bank-runs_pt`, `bitcoin-vs-cbdc_pt`, and `bitcoin-vs-gold_pt`.

To avoid duplicate work, please leave a comment here about which pages you are working on.

**Comment by sovenor** [comment]
Thanks for your time on this!

**Comment by sovenor** [comment]
Please note I added a new step to the Translations guide that should be applied to each file:

At the top of the file, change the "last-updated" date to today's date in YYYY-MM-DD format. Change the "locale" to to the [2 letter language code](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) that corresponds to your language. You can also optionally add your name in the list of authors.

Apologies for the oversight! Please make this change on all of the json files.

**Reply by OsodracPT** [reply]
Sorted 👍

**Comment by sovenor** [comment]
@OsodracPT I noticed an error in my English json file for the home page. I had "home_link_title_art_1" and "home_link_title_art_2" with the same duplicate headlines by mistake.

Please update "home_link_title_art_2" in index_pt.json with the translated version of "Send a Bitcoin Postcard to someone you know"

Thank you!

**Reply by OsodracPT** [reply]
Going to submit a PR

**Comment by sovenor** [comment]
Hi, please note that there are 4 new translation strings that are now in "common_pt.json"

"common_self_custody": "SELF-CUSTODY",
"common_not_your_keys": "NOT YOUR KEYS",
"common_cold_wallet": "COLD WALLET",
"common_hot_wallet": "HOT WALLET"

You should see these at the end of the file when you pull the latest changes. Thanks!

**Comment by sovenor** [comment]
A new translation string has been added to index_pt.json:

"home_link_title_energy_6"

Please translate when you have a moment :)

**Comment by sovenor** [comment]
Two new translation strings have been added to the top of common_pt.json:

"common_footer_mission_1"
"common_footer_mission_2"

Please translate these when you are able. Thank you!

**Comment by sovenor** [comment]
New translation strings have been added to inflation_pt.json:

Line 11: "inflation_sign_got_inflation"
Line 12: "inflation_save_in_bitcoin"
Line 42: "inflation_usd_but_why"

There are also 2 new pages that don't necessarily need translation since the signs campaign is USA only:

signs_pt.json
sign-success_pt.json

Thanks for your continued contributions!

**Comment by sovenor** [comment]
Translation strings have been modified slightly in wallets_pt.json:

Line 11: "two simple questions" is now "these simple questions"
Line 13: "two simple questions" is now "these simple questions"

These new strings have also been added to the same file:

Lines 33 - 39 to add a section about recovery phrases / seed phrases.

Please translate these if you have some spare time. Thank you!

**Reply by OsodracPT** [reply]
Done! PR sent.
